lib/monitoring/addon/components/cluster-dashboard/component.js

```diff
@@ -0,0 +1,61 @@
+import C from 'ui/utils/constants';
+import Component from '@ember/component';
+import { get, computed, observer, setProperties } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import layout from './template';
+
+export default Component.extend({
+  intl:        service(),
+  scope:       service(),
+  grafana:     service(),
+  globalStore: service(),
+  router:      service(),
+
+  layout,
+
+  nodes:             null,
+  components:        null,
+  monitoringEnabled: alias('scope.currentCluster.enableClusterMonitoring'),
+  componentStatuses: alias('scope.currentCluster.componentStatuses'),
+
+
+  init() {
+    this._super(...arguments);
+    this.setComponents();
```

Comment:
Instead of creating the init to to fetch the observer you can wrap the observer in an `on` method from the `evented` ember lib, passing it the 'init' sig. This will accomplish the same effect and is a little easier to understand imo.